onnx/defs/nn/defs.cc

```diff
@@ -1757,6 +1757,104 @@ ONNX_OPERATOR_SET_SCHEMA(
           propagateShapeAndTypeFromFirstInput(ctx);
         }));
 
+
+
+static const char* GroupNormalization_ver11_doc = R"DOC(
+Carries out group normalization as described in the paper
+https://arxiv.org/abs/1803.08494. 
```

Comment:
Could you please add some equations to describe this operator's behavior? Copying-and-pasting from the paper is fine as long as all symbols are well-defined.

Reply:
I am not certain what information from the paper you are requesting.